conman/redirects/models.py

```diff
@@ -12,11 +12,13 @@ class RouteRedirect(Route):
 
     This model holds the data required to make that connection.
     """
-    target = models.ForeignKey('routes.Route', related_name='+')
+    target = models.ForeignKey('routes.Route', related_name='+', on_delete=models.CASCADE)
     permanent = models.BooleanField(default=False)
 
     view = views.RouteRedirectView.as_view()
 
+    base_objects = models.Manager()
```

Comment:
It's not so much that we don't *want* to inherit it -- more that I'm not sure if we *need* to. I'm very much looking forward to the behaviour coming in django 2.0.